pages/k8s/1.28/release-notes.md

```diff
@@ -0,0 +1,88 @@
+
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "1.28 Release notes"
+  description: Release notes for Charmed Kubernetes
+keywords: kubernetes, release, notes
+tags: [news]
+sidebar: k8smain-sidebar
+permalink: 1.28/release-notes.html
+layout: [base, ubuntu-com]
+toc: False
+---
+# 1.28
+
+### August 16, 2023 - `charmed-kubernetes --channel 1.28/stable`
+
+The release bundle can also be [downloaded here](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.28/bundle.yaml).
+
+## What's new
+
+### Broader adoption of ops charming
+
+We're pleased to announce a number charms have been refactored from the `reactive` and `pod-spec` styles to `ops` style.  These uplifts were taken in an effort to have access to charm libraries, better support from juju, and better community support when demonstrating charming.  Some of the uplifted charms include:
+
+* calico
+* metallb
+
+And coming soon are a more charms like
+* kubeapi-load-balancer
+* kubernetes-worker
+* kubernetes-control-plane
+
+### Ceph CSI adds CephFS
+
+Providing much needed feature parity to the aging cdk-addons, the ceph-csi charm now supports operations of ceph-fs along with the already supported ceph-rdb.
+
```

Comment:
@charmed-kubernetes/canonical-k8s-team What other things would you like to highlight since release-1.27